### ISSUE TITLE
Pass UUID to KVM instances

### ISSUE THREAD

**Issue by GoogleCodeExporter** [olopez32/ganeti#605]
```
Feature request from a user on the mailing list:

"My issue is that with gnt-instance info, I can see that an instance UUID has 
been assigned, yet from within the instance itself it's not seeing the UUID. I 
looked and KVM can accept UUID as a command line argument. It seems that Ganeti 
is not passing this UUID through to the instance/KVM.

The reason this is an issue for me is we use SpaceWalk (locally hosts RHN, 
basically) and that needs all systems registered with it to have a UUID.

I'm running Ganeti 2.6.2 on Centos 6.4 nodes with Centos 6.4 instances."


More infos also in this thread: 
https://groups.google.com/forum/#!topic/ganeti/18EXjmecp5E
```

Original issue reported on code.google.com by `mtart...@google.com` on 4 Nov 2013 at 8:03
